### ISSUE TITLE
Incorrectly marking conflict in methods.

### ISSUE THREAD

**Issue by dryobates** [gforcada/flake8-builtins#62]
```python
# test_set.py
class MyClass:
    def set(self, value):
        pass


instance = MyClass()
instance.set("does not colide with builtins")
```

```
$ flake8 test_set.py
test_set.py:2:5: A003 "set" is a python builtin, consider renaming the class attribute
```

**Comment by dryobates** [comment]
Sorry for invalid issue. Just found A003.
